test/podman_commit.bats

```diff
@@ -13,109 +13,85 @@ function setup() {
 }
 
 @test "podman commit with change flag" {
-    run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} run -d --name my_ctr ${FEDORA_MINIMAL} sleep 6000"
+    ${PODMAN_BINARY} ${PODMAN_OPTIONS} run -d --name my_ctr ${FEDORA_MINIMAL} sleep 6000
+    run ${PODMAN_BINARY} ${PODMAN_OPTIONS} commit --change LABEL=image=blue my_ctr image-committed
     echo "$output"
     [ "$status" -eq 0 ]
-    run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} commit --change LABEL=image=blue my_ctr image-committed"
+    run bash -c ${PODMAN_BINARY} ${PODMAN_OPTIONS} inspect image-committed | grep blue"
     echo "$output"
     [ "$status" -eq 0 ]
-    run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} inspect image-committed | grep blue"
-    echo "$output"
-    [ "$status" -eq 0 ]
-    run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} rmi image-committed"
-    echo "$output"
-    [ "$status" -eq 0 ]
-    run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} stop my_ctr"
+    run "${PODMAN_BINARY} ${PODMAN_OPTIONS} rmi image-committed
     echo "$output"
     [ "$status" -eq 0 ]
+    ${PODMAN_BINARY} ${PODMAN_OPTIONS} stop my_ctr
 }
 
 @test "podman commit with pause flag" {
-    run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} run -d --name my_ctr ${FEDORA_MINIMAL} sleep 6000"
-    echo "$output"
-    [ "$status" -eq 0 ]
-    run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} commit --pause=false my_ctr image-committed"
+    ${PODMAN_BINARY} ${PODMAN_OPTIONS} run -d --name my_ctr ${FEDORA_MINIMAL} sleep 6000
+    run ${PODMAN_BINARY} ${PODMAN_OPTIONS} commit --pause=false my_ctr image-committed
     echo "$output"
     [ "$status" -eq 0 ]
     run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} images | grep image-committed"
     echo "$output"
     [ "$status" -eq 0 ]
-    run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} rmi image-committed"
-    echo "$output"
-    [ "$status" -eq 0 ]
-    run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} stop my_ctr"
+    run ${PODMAN_BINARY} ${PODMAN_OPTIONS} rmi image-committed
     echo "$output"
     [ "$status" -eq 0 ]
+    run ${PODMAN_BINARY} ${PODMAN_OPTIONS} stop my_ctr
 }
 
 @test "podman commit non-running container" {
-    run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} create --name my_ctr ${FEDORA_MINIMAL} ls"
-    echo "$output"
-    [ "$status" -eq 0 ]
-    run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} commit my_ctr image-committed"
+    ${PODMAN_BINARY} ${PODMAN_OPTIONS} create --name my_ctr ${FEDORA_MINIMAL} ls
+    run ${PODMAN_BINARY} ${PODMAN_OPTIONS} commit my_ctr image-committed
     echo "$output"
     [ "$status" -eq 0 ]
     run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} images | grep image-committed"
     echo "$output"
     [ "$status" -eq 0 ]
-    run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} rmi image-committed"
-    echo "$output"
-    [ "$status" -eq 0 ]
-    run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} rm my_ctr"
+    run ${PODMAN_BINARY} ${PODMAN_OPTIONS} rmi image-committed
```

Comment:
Just curious why you removed the tests for the stops, runs, creates from this module but not the others?   Personally I would have kept them here, but shrug.

Reply:
nothing particular ... i didnt remove any tests.  in cases where the commands results are not explicitly tied to what is being tested, it doesnt need to be preceeded with a run command,  nor its output or status checked.  for example, if a test of mine runs a container first thing so I can stop it, technically i dont need to preceed that container run with 'run'.